### PR TITLE
Run tests on python 3.13

### PR DESCRIPTION
**Pull request by jairhenrique**

### CODE REVIEW

**Review by sisp — COMMENTED** [review]
Thanks for enabling CI for Python 3.13, @jairhenrique! :pray: Just one question: Why did you re-lock all dependencies? It seems the tests were passing without a re-lock as well. Updating the lock file should be a separate PR IMO.

**Comment by jairhenrique** [comment]
@sisp I run `poetry lock` because tests breaks with `coverage==7.0.5`. After lock only one matrix fail with python 3.13 `build (windows-latest, 3.13)`.

You can see more details here:
- Windows fail https://github.com/jairhenrique/copier/actions/runs/11149852649
- Coverage fail https://github.com/jairhenrique/copier/actions/runs/11149733141

**Comment by sisp** [comment]
I see. Would you mind only updating the locked version for `coverage` instead of refreshing all locks?

```sh
poetry update coverage
```

**Comment by sisp** [comment]
It seems some other dependencies need to be updated as well.

**Comment by jairhenrique** [comment]
> It seems some other dependencies need to be updated as well.

I found it, is a `nodeenv` dependency. Updated!
Now to pass all tests we need a new version of `pywin32`. The project already supports python 3.13, but a new version has not been generated. https://github.com/mhammond/pywin32/pull/2260

**Comment by yajo** [comment]
seems like it's available already.
https://github.com/mhammond/pywin32/blob/842fbef58770d8c3a634b2e6033b95f80d19cbd3/CHANGES.txt#L32

could you update that dependency too please?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
```
Comment (sisp) [inline .github/workflows/ci.yml:51]: Python 3.13 has been released, so we don't need to enable pre-releases anymore.

Comment (pawamoy) [reply .github/workflows/ci.yml]: This will be useful for 3.14 anyway, no? Not a huge deal, we can add it back later.

**Comment by sisp** [comment]
It seems GitHub hasn't added Python 3.13 to their CI runners. :see_no_evil: We'll have to wait then ...

**Comment by jairhenrique** [comment]
> It seems GitHub hasn't added Python 3.13 to their CI runners. 🙈 We'll have to wait then – or revert to `prereleases: true` .

I revert the commit

**Comment by sisp** [comment]
I think CI on Windows with Python 3.13 is failing now because the locked `regex` version has no prebuilt wheel for Python 3.13 yet. Would you mind trying to update the lock file with `poetry update regex`?

**Review by sisp — APPROVED** [review]
LGTM! :tada: Thanks! :pray:

**Comment by sisp** [comment]
@yajo Could you please configure the CI jobs for Python 3.13 to be required? It's something we can't configure via PR.

**Comment by yajo** [comment]
Done, thanks

**Pull request merged**